### ISSUE TITLE
Want to mask only the peak of each island

### ISSUE THREAD

**Issue by ktrehaeven** [ratt-ru/breizorro#32]
Hi

Is there a way of masking only the peak pixel of an island instead of all its significant pixels?

**Comment by o-smirnov** [comment]
Not that I know of, but it should be very simple to add an option like that, feel free to make a PR. :)

What do you need such a feature for, anyway?

**Comment by ktrehaeven** [comment]
I want to see if subtracting only the peak pixel during vis plane subtraction improves the oversubtraction of the compact sources I'm seeing. Or perhaps just the pixels above the FWHM.

**Comment by o-smirnov** [comment]
Yeah I don't think a single pixel will work -- if a compact source straddles two pixels (as it almost always will), you want to be cleaning both. So maybe peak pixel + 1 pixel dilation.
